src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Proxy agent chunk requests and analytics data
+tags:
+  - Browser monitoring
+  - Proxy
+metaDescription: "The agent can be configured to fetch its code and send analytics through a proxy server instead of New Relic's endpoints."
+---
+
+From [version 1.240.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, the browser agent officially allows a proxy url to be set when determining either or both of:
+
+1. Where to fetch or lazy-load agent code. (Default: js-agent.newrelic.com CDN)
+2. Where to send analytics. (Default: bam.nr-data.net)
+
+Today, many ad-blockers and security VPNs out there block client requests to third-party domains including ours, and as the number of internet users opting into ad-blockers increase, we continue to lose observability on those devices. One solution would be to proxy those requests though an unblocked middleman. For instance, `abc.com` can send its visitor analytics through its `nrbeacon.abc.com` proxy which would not be blocked by anti-adware being that it's considered a first-party (sub)domain.
+
+<Callout variant="important">
+  This feature is currently available for those using the copy-and-paste _or_ NPM installation methods. There is no UI configuration at this time, and APM auto-injected agents are not able to use this.
+</Callout>
+
+## How to use a proxy [#use]
+
+On the agent side, it looks at two configuration properties when determining where to fetch chunks or send data per above.
+
+For example, using copy and paste:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>,
+    proxy: {
+        assets: <URL to fetch agent chunks>,
+        beacon: <URL to send analytics>
+    }
+}
+```
+
+For NPM, similarly just pass the `init` object with a proxy as part of any other configurations:
+
+```js
+new BrowserAgent({ init: { proxy: <object similar to above> } })
+```
+
+`init.proxy.assets` is used verbatim in the copy-and-paste installation to fetch agent chunks. This implies you are hosting the file somewhere instead it being retrieved via our CDN. 
+
+* For example, the v1.240.0 Pro+SPA loader will require `nr-spa.___-1.240.min.js` file.
+* If you are hosting the file on a `nrassets.abc.com` proxy, you must provide this value as the URL, `'https://nrassets.abc.com/'`, including the scheme and ending slash.
+
+This is not relevant to NPM installed agents, as the main host project's bundler should take care of resources. `init.proxy.beacon` is used by both installation to determine the host to which data such as page view timings are sent. 
+
+* For example, if those payloads are routed through a `nrbeacon.abc.com` proxy, then that is the exact string value to use. 
+* Note that the scheme (e.g, `'https://'`) and ending slash **should be omitted** for this property.
+
+```js
+NREUM.init = {
+    proxy: {
+        assets: 'https://exampleproxy.com/',
+        beacon: 'exampleproxy.com'
```

Comment:
our example should probably show that the assets and beacon are meant to be separate paths

Reply:
They don't have to be but it would make setting up and managing the proxy a lot simpler for the customer.

Reply:
In fact I would probably put that into a callout....our recommendation to use different paths for each item.